packages/grid/src/CellRenderer.ts

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable class-methods-use-this */
+import { getOrThrow } from '@deephaven/utils';
+import { isExpandableGridModel } from './ExpandableGridModel';
+import { VisibleIndex, Coordinate, BoxCoordinates } from './GridMetrics';
+import { GridRenderState } from './GridRendererTypes';
+import { GridColor } from './GridTheme';
+
+export type CellRendererType = 'text' | 'databar';
```

Comment:
We should be consistent, name it `'dataBar'` (since data bar is two words, and we're using that casing everywhere else).
Also, upon reflection, I like the name `CellRenderType` better.
```suggestion
export type CellRendererType = 'text' | 'dataBar';
```